docs/administration/configuration/config-file-reference.md

```diff
@@ -325,6 +325,10 @@ Some of the properties that work with live reloading:
 
 ### Security HTTP Headers
 
+:::warning
+The HTTP header 'XSS-Protection' has been deprecated by mordern browsers and it use can introduce additional security issues on the client side of the application. So Rundeck will not support this header no more since version 4.3.0 .
```

Comment:
Replace 2nd Sentence:
"Rundeck has deprecated this setting as of version 4.3.0."

Reply:
done